tests/www/test_views.py

```diff
@@ -85,10 +85,11 @@ def login(self):
     def logout(self):
         return self.client.get('/logout/')
 
-    def clear_table(self, model):
-        self.session.query(model).delete()
-        self.session.commit()
-        self.session.close()
+    @classmethod
+    def clear_table(cls, model):
+        cls.session.query(model).delete()
```

Comment:
I would prefer:
```suggestion
        with create_session() as session:
            session.query(model).delete()
```

Reply:
I guess since we're closing the session anyway that makes sense here.

Reply:
Done.